### ISSUE TITLE
On Unleashed Firmware - Malveke apps offered in the version 1.0

### ISSUE THREAD

**Issue by explit28** [EstebanFuentealba/MALVEKE-Flipper-Zero#9]
Hello @EstebanFuentealba I'm not sure if it is bug or feature, but on unleashed firmware the Flipper App on iPhone always offers me MALVEKE Apps in the Version 1.0, where 1.1 is out since couple of weeks.

Is it normal, or is it "bug" of unleashed Firmware?

Update 1
If i try to download ready-made fap-file for unleashed Firmware on your Github Page i'm offered Release 50.0

![Screenshot 2023-12-27 at 02-02-12 MALVEKE GB Live Camera - FlipC org](https://github.com/EstebanFuentealba/MALVEKE-Flipper-Zero/assets/11852348/87353f41-8671-4c99-807d-a0a450a72ba0)

I see that the file with the 50.0 Release differs from 1.0 which i get offered in the Flipper Zero App. So my question is which release should i use on the unleashed Firmware?

**Comment by RogueMaster** [comment]
Hi, @explit7 you are writing this issue here when it is clearly an issue with Unleashed not updating. You may want to request people from Unleashed update more often.

**Comment by explit28** [comment]
> Hi, @explit7 you are writing this issue here when it is clearly an issue with Unleashed not updating. You may want to request people from Unleashed update more often.

Thanks for pointing that out. @RogueMaster For me it was not really clear, so you wrote it here. @EstebanFuentealba Just close the issue, when its clear, that the problem is on unleashed side.

Thanks

Update: Also the unleashed guys confirmed, that this issue has nothing to do with unleashed firmware

**Comment by EstebanFuentealba** [comment]
Hello, there are two things to consider:

- The firmware displayed on flipc (50.0) is for Flipper Zero, which is different from the MALVEKE PCB firmware. Flipc will always install the most updated version of the apps (1.1) available in the repository.

- I haven't updated the version in the official catalog yet because I'm still waiting for confirmation regarding a bug fix related to the GAME BOY Camera and RGB #7 screens.

In summary, it's not an error from Unleashed or flipc; it's just that I haven't updated the official catalog yet 🤓.
